modules/core/arrow-instances/src/main/kotlin/arrow/instances/try.kt

```diff
@@ -107,8 +108,8 @@ interface TryTraverseInstance : Traverse<ForTry> {
     override fun <A, B> map(fa: TryOf<A>, f: kotlin.Function1<A, B>): Try<B> =
             fa.fix().map(f)
 
-    override fun <G, A, B> traverse(fa: TryOf<A>, f: kotlin.Function1<A, Kind<G, B>>, GA: Applicative<G>): Kind<G, Try<B>> =
-            fa.fix().traverse(f, GA)
+    override fun <G, A, B> Applicative<G>.traverse(fa: Kind<ForTry, A>, f: (A) -> Kind<G, B>): Kind<G, Try<B>> =
```

Comment:
We really need `sequence` everywhere because that is gonna be one of the most useful combinators ever that people are missing out and I get a lot of questions around `traverse` when they just want to thread the effects traversing over the identity